api/build.gradle

```diff
@@ -669,7 +669,7 @@ def db_port = System.getenv("DB_PORT")
 // See project.rb command: fetch-firecloud-user-profiles
 task fetchFireCloudUserProfile(type: JavaExec) {
```

Comment:
Not sure if this command is exist
<img width="632" alt="Screenshot 2023-04-27 at 9 51 20 AM" src="https://user-images.githubusercontent.com/34481816/234883140-5c6ec738-cec2-4b0c-87cc-146c9541e72e.png">

Reply:
Good catch. I won't address here to keep this PR as mechanical as possible, but maybe a follow-up. I tried the Gradle task on a clean branch and it failed, so this does need some attention.